devices/pcie/gpus.json

```diff
@@ -32,61 +32,211 @@
         "interface": "PCIe",
         "memory_size": "80Gi"
       },
-      "20f1": {
+      "20b2": {
         "name": "a100",
-        "interface": "PCIe",
-        "memory_size": "40Gi"
+        "interface": "SXM4",
+        "memory_size": "80Gi"
       },
       "20f3": {
         "name": "a800",
         "interface": "SXM4",
         "memory_size": "80Gi"
       },
-      "1eb8": {
-        "name": "t4",
+      "1d01": {
+        "name": "gt1030",
         "interface": "PCIe",
-        "memory_size": "16Gi"
+        "memory_size": "2Gi"
       },
-      "2489": {
-        "name": "rtx3060ti",
+      "1c81": {
+        "name": "gtx1050",
+        "interface": "PCIe",
+        "memory_size": "2Gi"
+      },
+      "1c83": {
+        "name": "gtx1050",
+        "interface": "PCIe",
+        "memory_size": "3Gi"
+      },
+      "1c82": {
+        "name": "gtx1050ti",
+        "interface": "PCIe",
+        "memory_size": "4Gi"
+      },
+      "1c02": {
+        "name": "gtx1060",
+        "interface": "PCIe",
+        "memory_size": "3Gi"
+      },
+      "1c04": {
+        "name": "gtx1060",
+        "interface": "PCIe",
+        "memory_size": "5Gi"
+      },
+      "1c03": {
+        "name": "gtx1060",
+        "interface": "PCIe",
+        "memory_size": "6Gi"
+      },
+      "1c06": {
+        "name": "gtx1060",
+        "interface": "PCIe",
+        "memory_size": "6Gi"
+      },
+      "1b81": {
+        "name": "gtx1070",
         "interface": "PCIe",
         "memory_size": "8Gi"
       },
-      "2204": {
-        "name": "rtx3090",
+      "1b82": {
+        "name": "gtx1070ti",
         "interface": "PCIe",
-        "memory_size": "24Gi"
+        "memory_size": "8Gi"
       },
-      "2216": {
-        "name": "rtx3080",
+      "1b80": {
+        "name": "gtx1080",
         "interface": "PCIe",
-        "memory_size": "10Gi"
+        "memory_size": "8Gi"
       },
-      "2684": {
-        "name": "rtx4090",
+      "1b06": {
+        "name": "gtx1080ti",
+        "interface": "PCIe",
+        "memory_size": "11Gi"
+      },
+      "1f95": {
+        "name": "gtx1650ti",
+        "interface": "PCIe",
+        "memory_size": "4Gi"
+      },
+      "2187": {
+        "name": "gtx1650super",
+        "interface": "PCIe",
+        "memory_size": "4Gi"
+      },
+      "2184": {
+        "name": "gtx1660",
+        "interface": "PCIe",
+        "memory_size": "6Gi"
+      },
+      "2182": {
+        "name": "gtx1660ti",
+        "interface": "PCIe",
+        "memory_size": "6Gi"
+      },
+      "1401": {
+        "name": "gtx960",
+        "interface": "PCIe",
+        "memory_size": "2Gi"
+      },
```

Comment:
Information about this hardware was not found, maybe it means `gtx950 2G with ID 1402`?
source https://www.techpowerup.com/vgabios/257069/257069